source/cloud-security/gcp/prerequisites/credentials.rst

```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
 
 To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as it will be used as the authentication method for the GCP module.
 
+Creating a custom role
+----------------------
+
+The ``gcp-bucket`` module requires permissions to access storage buckets and objects. These permissions were originally assigned using the **Storage Legacy Bucket Writer** role, but it is no longer available for selection, so it will be necessary to create a new one.
+
+To do so, follow these steps:
+
+1. Go to the **Roles** section and click on **Create Role**.
+2. Choose a name and click on **Add Permissions**.
+3. On the search bar, type **Storage Legacy Bucket Writer** and select the following permissions:
+
+   - ``storage.bucket.get``
+   - ``storage.objects.create``
+   - ``storage.objects.delete``
+   - ``storage.objects.list``
+
+4. Click on **Create**.
```

Comment:
In the previous suggestion the 4th point was below the image but it didn't look so good once compiled, so I included this point below the other ones

![screenshot](https://github.com/wazuh/wazuh-documentation/assets/51374959/1bc86cc8-ec27-4d27-b7e9-b5e313d9d4cc)